try:
    print(ok)
except(Exception):
    print("Invalid Syntax Caught through except")
finally:
    print("This is in Finally Block")